docs/guide/displaying_content_with_simple_templates.md

```diff
@@ -0,0 +1,96 @@
+# Content item rendering
```

Comment:
The title is a bit too close to "Content rendering" we have above, I'm thinking something line "Templating basics"?
(You could also change file name, as links will be broken here anyway)